src/views/Entries/LeadGroupedEntries/Entry/index.js

```diff
@@ -269,7 +276,8 @@ export default class Entry extends React.PureComponent {
                 <Widget
                     widget={widget}
                     widgetName={widgetId}
-                    widgetType={VIEW.list}
+                    wid
+                    getType={VIEW.list}
```

Comment:
I think we should also rename the prop `getType`

Reply:
I think it's `widgetType={VIEW.list}`